src/platform_impl/macos/window.rs

```diff
@@ -1235,13 +1240,13 @@ pub(super) fn get_ns_theme() -> Theme {
     }
 }
 
-fn set_ns_theme(theme: Theme) {
+fn set_ns_theme(theme: Option<Theme>) {
     let app = NSApp();
     let has_theme: bool = unsafe { msg_send![&app, respondsToSelector: sel!(effectiveAppearance)] };
     if has_theme {
         let name = match theme {
-            Theme::Dark => NSString::from_str("NSAppearanceNameDarkAqua"),
-            Theme::Light => NSString::from_str("NSAppearanceNameAqua"),
+            Some(Theme::Dark) | None => NSString::from_str("NSAppearanceNameDarkAqua"),
+            Some(Theme::Light) => NSString::from_str("NSAppearanceNameAqua"),
         };
         let appearance = NSAppearance::appearanceNamed(&name);
         app.setAppearance(&appearance);
```

Comment:
`NSApplication::setAppearance` is wrongly mapped, it should be (note the `Option`):
```rust
#[sel(setAppearance:)]
pub fn setAppearance(&self, appearance: Option<&NSAppearance>);
```

Which would allow you to reset back to the default of not having overridden the theme.

Reply:
thanks